ion/readerlocalsymboltable_test.go

```diff
@@ -0,0 +1,189 @@
+package ion
+
+import (
+	"testing"
+)
+
+func TestLocalSymbolTableAppend(t *testing.T) {
+	text := "$ion_symbol_table::" +
```

Comment:
Pro go tip: backtick-delimited ("raw") string literals are great for this kind of thing. See [here](https://github.com/amzn/ion-go/blob/master/ion/textwriter_test.go#L403) for an example.

Reply:
Thanks! , updated